tests.edn

```diff
@@ -9,7 +9,11 @@
                     {:id :integration
                      :ns-patterns ["rems"]
                      :test-paths ["src" "test"]
-                     :focus-meta [:integration]}]
+                     :focus-meta [:integration]}
+                    {:id :focused
```

Comment:
Should this be in some other pull request? Can't see anywhere focused appended elsewhere in this PR?

Reply:
The :focused tag is not meant to be committed. It's for easily choosing which subset of tests to run.

Reply:
Still don't get why it is added in this pull request.